src/Google.Cloud.Functions.Framework/ITypedFunction.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020, Google LLC
```

Comment:
nit: 2023, and maybe check elsewhere

Reply:
Thanks, will fix - it looks like we're missing headers in another couple of files too.